rtcpeerconnection.js

```diff
@@ -13,7 +13,9 @@ var shimIceGatherer = require('./rtcicegatherer');
 var shimIceTransport = require('./rtcicetransport');
 var shimDtlsTransport = require('./rtcdtlstransport');
 var getCommonCapabilities = require('./getcommoncapabilities');
-var writeMediaSection = require('./writemediasection');
+var writeMediaSection = require('./writemediasection').writeMediaSection;
```

Comment:
this is a bit ugly but will get better with ES6.

Reply:
not yet. And I doubt it makes sense to port this due to the limited remaining useful lifetime :-/